scripts/expand_alignment_duplicates.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
Nit: For module level comments, it can be helpful to include a 1 sentence summary of the purpose of the module so the reader can quickly understand what the module is for.

Not sure how this will play with the `__doc__` call you use in some of these scripts though. Feel free to keep the single paragraph version if you think that is nicer for the `__doc__` call.